charts/graph-kubernetes/templates/cronjob.yaml

```diff
@@ -61,3 +239,32 @@ spec:
           tolerations:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          terminationGracePeriodSeconds: 10
```

Comment:
Think default is 30, particular reason you wanted it shorter?

Reply:
Just to keep it more in line with how often we collect the container metrics (every 10 seconds or so).